src/Bicep.Cli/Arguments/TestArguments.cs

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Bicep.Core.FileSystem;
+using System;
+using System.IO;
+
+namespace Bicep.Cli.Arguments
+{
+    public class TestArguments : ArgumentsBase
+    {
+        public TestArguments(string[] args) : base(Constants.Command.Test)
+        {
+            for (var i = 0; i < args.Length; i++)
+            {
+                switch (args[i].ToLowerInvariant())
+                {
+                    case "--stdout":
```

Comment:
It's not clear to me what some of these flags would do for tests (`--stdout`, `--outfile`, `--outdir`). Since the command is hidden from regular users, I think it's fine to leave these here for now, but we should think about removing things that don't make sense once we do the implementation.

Reply:
Sounds good, 
I will delete them in the implementation PR